trade-module/src/main/java/io/edpn/backend/trade/adapter/web/TradeModuleController.java

```diff
@@ -42,32 +50,32 @@ public List<RestValidatedCommodityDto> findByFilter(RestFindCommodityFilterDto f
         return findValidatedCommodityByFilterUseCase
                 .findByFilter(findCommodityRequest)
                 .stream()
-                .map(x -> (RestValidatedCommodityDto)x)
+                .map(x -> (RestValidatedCommodityDto) x)
                 .toList();
     }
 
     @GetMapping("/commodity/{displayName}")
     public Optional<RestValidatedCommodityDto> findByName(@PathVariable String displayName) {
         return findValidatedCommodityByNameUseCase
                 .findByName(displayName)
-                .map(x -> (RestValidatedCommodityDto)x);
+                .map(x -> (RestValidatedCommodityDto) x);
     }
-    
-    @GetMapping("/locate-commodity/filter")
-    public List<RestLocateCommodityDto> locateCommodityWithFilters(RestLocateCommodityFilterDto locateCommodityFilterDto){
+    @ApiResponses({
```

Comment:
This endpoint does not work. [] do not encode properly in tomcat 8.5+, http encrypts [ = %5B and ] = %5D
Find a way to fix if we are going to maintain this structure for the endpoint